website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,385 @@
+
+To view the generated values at a later time, navigate to **Providers** and select the provider by name, then click the **Edit** button.
+
+:::
+
+#### 3.3. Assign scope mappings
+
+Under the **Advanced protocol settings**, add the newly created scope mappings to the provider, as well as the default `profile` scope:
+
+- [x] `Apple Business Manager ssf.manage`
+- [x] `Apple Business Manager ssf.read`
+- [x] `Apple Business Manager profile`
+- [x] `authentik default OAuth Mapping: OpenID 'profile'`
+
+Click **Finish** and confirm that `Apple Business Manager` is listed in the provider overview.
+
+:::tip OpenID Configuration URL
+
+From the provider overview page, copy the URL in the **OpenID Configuration URL** field to your text editor. You'll need this when configuring Apple Business Manager.
+
+You can always find your provider's generated URLs by navigating to **Providers** and selecting the provider by name. The URLs are listed under the **Overview** tab.
+
+:::
+
+### 4. Create Shared Signals Framework provider
+
+While the OIDC provider handles the authentication flow, you'll need to create a [Shared Signals Framework provider](https://docs.goauthentik.io/docs/add-secure-apps/providers/ssf/) to handle the backchannel communication between authentik and Apple Business Manager.
+
+From the authentik Admin interface, navigate to **Applications** -> **Providers** and click **Create**. Select **Shared Signals Framework Provider** and continue with the following form values:
+
+| Field           | Value                        |
+| --------------- | ---------------------------- |
+| Name            | `Apple Business Manager SSF` |
+| Signing Key     | `[Your Signing Key]`         |
+| Event Retention | `days=30`                    |
+
+Click **Finish** and confirm that the new SSF provider is listed in the overview.
+
+:::tip A Blank SSF Config URL is expected
+
+Keep in mind the **SSF Config URL** will be blank until the SSF provider is assigned to an application as a backchannel provider. We'll return to collect this URL after creating the application.
+
+:::
+
+### 5. Create application
+
+From the authentik Admin interface, navigate to **Applications** -> **Applications** and click **Create**. Continue with the following form values:
+
+| Field                | Value                      |
+| -------------------- | -------------------------- |
+| Name                 | Apple Business Manager     |
+| Slug                 | `abm`                      |
+| Provider             | Apple Business Manager     |
+| Backchannel Provider | Apple Business Manager SSF |
+
+Click **Create** and confirm that the application is listed in the overview page.
+
+:::tip Your SSF Config URL is now available
+
+Assigning an SSF provider to a application will generate **SSF Config URL**.
+
+Navigate to **Providers** -> **Apple Business Manager SSF** and copy the `SSF Config URL` value to your text editor.
+
+:::
+
+---
+
+## Configuration checklist
```

Comment:
I do see the usefulness of  checklist but don't think it is needed here. We want the docs to be lean, and these steps repeat the steps above. And... there is still a LOT of content still to go. :-)